internal/project/project_test.go

```diff
@@ -0,0 +1,204 @@
+package project
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/BurntSushi/toml"
+)
+
+func TestDecodeSimple(t *testing.T) {
```

Comment:
Think we should keep tests consistent in using `spec`.

Reply:
@jromero remind me, are we pretty much using `spec` across the board for all CNB sponsored repos?

Reply:
@jromero I went ahead and updated the test suite to be `spec` based.  Let me know if I missed anything else.  Thanks.